diff --git a/build/generate-command.js b/build/generate-command.js
index 5d20cc15d3f5dbd3025d2b80bbf0eacb3ce58783..e250f88f0dbc7342baa9e7bf3470aa428c5dde7d 100644
--- a/build/generate-command.js
+++ b/build/generate-command.js
@@ -1,7 +1,7 @@
 import process from 'node:process';
 import { render } from 'ink';
 import React from 'react';
-import { StatusMessage } from '@inkjs/ui';
+import { Text } from 'ink';
 import { fromZodError } from 'zod-validation-error';
 import generateOptions from './generate-options.js';
 import generateArguments from './generate-arguments.js';
@@ -44,7 +44,7 @@ const generateCommand = (commanderCommand, pastelCommand, { appComponent }) => {
                     parsedOptions = result.data ?? {};
                 }
                 else {
-                    render(React.createElement(StatusMessage, { variant: "error" }, fromZodError(result.error, {
+                    render(React.createElement(Text, { color: "red" }, "❌ " + fromZodError(result.error, {
                         maxIssuesInMessage: 1,
                         prefix: '',
                         prefixSeparator: '',
@@ -60,7 +60,7 @@ const generateCommand = (commanderCommand, pastelCommand, { appComponent }) => {
                     arguments_ = result.data ?? [];
                 }
                 else {
-                    render(React.createElement(StatusMessage, { variant: "error" }, fromZodError(result.error, {
+                    render(React.createElement(Text, { color: "red" }, "❌ " + fromZodError(result.error, {
                         maxIssuesInMessage: 1,
                         prefix: '',
                         prefixSeparator: '',
